### ISSUE TITLE
Open Health Imaging Foundation viewer not displaying the file in the list

### ISSUE THREAD

**Issue by cyberuna** [microsoft/dicom-server#405]
**Describe the bug**
I uploaded a dcm file of cardiomelagy (
[dicom_00000001_000.zip](https://github.com/microsoft/dicom-server/files/5433879/dicom_00000001_000.zip)
). 
In the App Server, in Open Health Imaging Foundation viewer, I don't this in the list. Tried changing the dates range to 12/31/9999 as the image Study Date says 12-Dec-99.
But if I add the StudyInstanceUID at the end of the URL, it works.

**To Reproduce**
1. Upload the attached dcm file
2. Open DICOM App Service, check if the file shows in the list
3. Add the StudyInstanceUID to the App Service URL

**Expected behavior**
Expected the file to show up in the list when I change the end date to 12/31/9999.

**Actual behavior**
The file did not show up in the list.

**Comment by abdullah248** [comment]
Hi,
After investigating it seems that the issue is that the file attached does not have a Study Date so it does not show up when filtered by date in the Open Health Imaging Foundation viewer. If you remove the date filter then the file will show up and in the date field it shows invalid date.

**Comment by StevenBorg** [comment]
@cyberuna, I'm planning on closing this issue.  I opened up the linked DICOM file and used the DICOM Tag Dump VS Code extension to examine the tags.  Here's what I got.  As noted by @abdullah248 above, there is no StudyDate in the DICOM metadata. 

(0002,0000) UL FileMetaInformationGroupLength = 204
(0002,0001) OB FileMetaInformationVersion = <bin: 0x0001>
(0002,0002) UI MediaStorageSOPClassUID = 1.2.840.10008.5.1.4.1.1.7
(0002,0003) UI MediaStorageSOPInstanceUID = 1.3.6.1.4.1.11129.5.5.153751009835107614666834563294684339746480
(0002,0010) UI TransferSyntaxUID = 1.2.840.10008.1.2
(0002,0012) UI ImplementationClassUID = 1.2.826.0.1.3680043.8.498.1
(0002,0013) SH ImplementationVersionName = PYDICOM 1.2.0 
(0008,0016) UI SOPClassUID = 1.2.840.10008.5.1.4.1.1.7
(0008,0018) UI SOPInstanceUID = 1.3.6.1.4.1.11129.5.5.153751009835107614666834563294684339746480
(0008,0060) CS Modality = DX
(0008,1030) LO StudyDescription = Cardiomegaly
(0010,0020) LO PatientID = 1 
(0010,0040) CS PatientSex = M
(0010,1010) AS PatientAge = 58
(0020,000D) UI StudyInstanceUID = 1.3.6.1.4.1.11129.5.5.111396399361969898205364400549799252857604
(0020,000E) UI SeriesInstanceUID = 1.3.6.1.4.1.11129.5.5.195628213694300498946760767481291263511724
(0028,0002) US SamplesPerPixel = 1
(0028,0004) CS PhotometricInterpretation = MONOCHROME2
(0028,0010) US Rows = 1024
(0028,0011) US Columns = 1024
(0028,0100) US BitsAllocated = 8
(0028,0101) US BitsStored = 8
(0028,0102) US HighBit = 7
(0028,0103) US PixelRepresentation = 0
(7FE0,0010) OW|OB PixelData = <binary data of length: 1048576>

To validate it wasn't a different issue, I opened the DCM file with pydicom, added a StudyDate, and resaved the file. It then showed up in a date filtered query in the OHIF viewer.  

@cyberuna, let me know if I've missed something, and I'll reopen the Issue!
